### ISSUE TITLE
beakerlib test executed on the host

### ISSUE THREAD

**Issue by t184256** [psss/tmt#40]
When I `tmt run --debug discover provision -h vagrant prepare execute report finish` a test, it provisions a Vagrant VM and then executes `nohup ...` on the host itself and not the VM.

relevant log fragment:
```
    execute
        workdir: /var/tmp/tmt/run-076/plans/example/execute
        how: beakerlib
        Copy '/home/asosedki/code/tmt/tmt/steps/execute/run.sh' to '/var/tmp/tmt/r
un-076/plans/example/execute'.
            Run command '('vagrant', 'rsync')'.
            out: ==> default: Rsyncing folder: /var/tmp/tmt/run-076/plans/example/
 => /var/tmp/tmt/run-076/plans/example
        Run command 'nohup /var/tmp/tmt/run-076/plans/example/execute/run.sh -v /v
ar/tmp/tmt/run-076/plans/example beakerlib /var/tmp/tmt/run-076/plans/example/exec
ute/stdout.log /var/tmp/tmt/run-076/plans/example/execute/stderr.log'.
        err: nohup: ignoring input
            Run command '('vagrant', 'plugin', 'install', 'vagrant-rsync-back')'.
```

**Comment by pvalena** [comment]
Vagrant should not be `how` ... it's just a translation/abstraction layer to `libvirt`, `hyper-v`, `VirtualBox`, `openstack`, `docker` and any other Cloud or platform you can think of.

`rsync-back` plugin is merely an Rsyncing the workdir back from guest to host. (Friendly error messages is in PR #30.) What you need to simply remove system-installed (or any other plugin) `vagrant-libvirt` do is due to an upstream:downstream dependency conflict.

Unfortunatelly I've not got around to package vagrant-rsync-back to Fedora yet.

**Comment by pvalena** [comment]
The issue reported should be fixed already in #30.

**Comment by psss** [comment]
Yes, confirmed. This has been fixed by #30 and is now merged in master.
